docs/get-started/svelte-webpack5.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Storybook for Svelte & Webpack
+---
+
+export const SUPPORTED_RENDERER = 'svelte';
+
+Storybook for Svelte & Webpack is a [framework](../contribute/framework.md) that makes it easy to develop and test UI components in isolation for applications using [Svelte](https://svelte.dev/) built with [Webpack](https://webpack.js.org/).
+
+<If notRenderer={SUPPORTED_RENDERER}>
+
+<Callout variant="info">
+
+Storybook for Svelte & Webpack is only supported in [Svelte](?renderer=svelte) projects.
+
+</Callout>
+
+<!-- End non-supported renderers -->
+
+</If>
+
+<If renderer={SUPPORTED_RENDERER}>
+
+## Requirements
+
+- Svelte ≥ 3.0
```

Comment:
Curious where you got 3.0 from?

```suggestion
- Svelte ≥ 4.0
```

Reply:
Oh I guess Storybook 7 supports Svelte 3. I would expect this content to be versioned though like all the other docs?